content/en/blog/2020/wasmhub-istio/index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Extended and Improved WebAssemblyHub to Bring the Power of WebAssembly to Envoy and Istio
+subtitle: A place to build, publish, share, and deploy WebAssembly Envoy extensions
+description: Community partner tooling of Wasm for Istio by Solo.io.
+publishdate: 2020-03-13
+attribution: "Idit Levine (Solo.io)"
+keywords: [wasm,extensibility,alpha,performance,operator]
+---
+
+*[Originally posted on the Solo.io blog](https://www.solo.io/blog/an-extended-and-improved-webassembly-hub-to-helps-bring-the-power-of-webassembly-to-envoy-and-istio/)*
+
+As you look to adopt Envoy-based infrastructure like Istio to help them solve challenges with microservices communication, they inevitably find themselves needing to customize some part of that infrastructure to fit within their organization's constraints. [WebAssembly (Wasm)](https://webassembly.org/) has emerged as a safe, secure, and dynamic environment for platform extension.
+
+Last week's [announcement of Istio 1.5](/blog/2020/wasm-announce/), lays the foundation for bringing WebAssembly to the popular Envoy proxy. [Solo.io](https://solo.io) is collaborating with Google and the Istio community to simplify the overall experience of creating, sharing, and deploying WebAssembly extensions to Envoy and Istio. It wasn't that long ago that Google and others laid the foundation for containers, and Docker built a great user experience to make it consumable. Similarly, this effort makes Wasm consumable by building the best user experience for WebAssembly on Istio.
+
+Back in December 2019, Solo.io began an effort to provide a great developer experience for WebAssembly with the announcement of WebAssembly Hub. The WebAssembly Hub allows developers to very quickly spin up a new WebAssembly project in C++ (we're expanding this language choice, see below), build it using Bazel in Docker, and push it to an OCI-compliant registry. From there, operators had to  pull the module, and configure Envoy proxies themselves to load it from disk. Beta support in [Gloo, an API Gateway built on Envoy](https://docs.solo.io/gloo/latest/) allows you to declaratively and dynamically load the module, the Solo.io team wanted to bring the same effortless and secure experience to other Envoy-based frameworks as well - like Istio. 
```

Comment:
Remove the extra space char at the end of this line. Also several others (see lint errors).